server/http/src/test_utils.rs

```diff
@@ -59,3 +59,13 @@ pub fn assert_error_response(
         .expect("GraphQL \"errors\" field must be a vector")
         .clone()
 }
+
+pub fn assert_expected_headers(response: &Response<Body>) {
```

Comment:
If you annotate this function with `#[track_caller]`, a failing test will point to where this function is called, rather than at the `assert_eq!` in here. Not a huge deal, but can make life a little easier when things go wrong

Reply:
That's awesome! I didn't know that.
I'm sending a new commit with that addition.